lib/dor/services/client/object_version.rb

```diff
@@ -52,7 +52,7 @@ def openable?(**params)
         # rubocop:enable Metrics/MethodLength
 
         # Open new version for an object
-        # @param params [Hash] optional params (see dor-services-app)
+        # @param params [Hash] required (:description, :significant) and optional params (see dor-services-app)
```

Comment:
This is not a `Hash`, it is an argument list, so it should have one kwarg per `@param`.

Reply:
right-o!  good save.